packages/material-ui/src/Backdrop/Backdrop.js

```diff
@@ -39,7 +39,7 @@ const Backdrop = React.forwardRef(function Backdrop(props, ref) {
           },
           className,
         )}
-        aria-hidden="true"
+        aria-hidden
```

Comment:
I have looked at it on reach-ui side, they use both interchangeably. How does it work? 
`aria-hidden` -> shorthand notation expension`aria-hidden={true}` -> DOM string casting `aria-hidden="true"`